index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>BADDEST BLUES DESIGN | A PORTFOLIO WEBSITE.</title>
+	<link href="https://fonts.googleapis.com/css?family=Cutive+Mono" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="/css/styles.css">
+</head>
+<body>
+
+	<h1>BADDEST BLUES DESIGN | A PORTFOLIO WEBSITE.</h1>
+	<div class="isabel"></div>
+
+	<article>
+		<h2>ABOUT BADDEST BLUES DESIGN:</h2>
+		<p class="para-column">Organic small batch aesthetic crucifix intelligentsia. Fixie paleo meggings, edison bulb knausgaard fanny pack asymmetrical pabst selvage readymade polaroid typewriter disrupt. Gastropub coloring book godard, enamel pin VHS pinterest trust fund artisan gentrify flannel pickled. You probably haven't heard of them etsy pinterest helvetica, knausgaard flexitarian bespoke sartorial pitchfork affogato photo booth bicycle rights humblebrag try-hard ramps. Iceland biodiesel farm-to-table mlkshk. Meh fanny pack squid echo park hashtag. Fingerstache ethical austin chillwave lyft.</p>
+	</article>
+
+	<article>
+		<h2>LINKS TO COOL SITES:</h2>
+		<p class="links-para"><a href="https://hipsum.co" target="_blank">Hipster Ipsum</a>
+		<br><a href="https://developer.mozilla.org/en-US/" target="_blank">Mozilla Developer Network</a>
+		<br><a href="https://creativemarket.com/" target="_blank">Creative Market</a></p>
+	</article>
+
+	<article>
+		<h2>PROJECTS IN THE WORKS:</h2>
+		
+			<section>
+			<h3>PROJECT #1:</h3>
```

Comment:
are you going to have screen shots of projects next to the text? (the correct answer here is yes)  if so stick a fake project screenshot into the place you would put it.